### PR TITLE
Created Starbucks-Landing-page

### PR DESCRIPTION
**Pull request by Sneha123-zudo**
# Pull Request for WebMasterLog 💡

## Issue Title: Starbucks Landing Page

- **Name of the project** : Starbucks Landing Page
- **Category:**  Frontened Project(HTML , CSS)
- **Name:** Sneha Mishra
- **GitHub ID:** https://github.com/Sneha123-zudo
- **Idenitfy yourself:** Contributor under JWOC

Closes: #97 

### Describe the add-ons or changes you've made 📃
Added a new project.

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

1. 
image

2.
image

3.
image

4.
image

5.
image
 
6.
image

7.
image

## Note to reviewers 📄

Please review my pull reuest

### CODE REVIEW

File: Front-end-Projects/Basic/Starbucks-Landing-page/README.md
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Basic/Starbucks-Landing-page/README.md:1]: Follow this updated [README](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/ccfcdca6d08efd69961187762ea2707d02def086/.github/PROJECT-README-TEMPLATE.md) template.

File: Front-end-Projects/Basic/Starbucks-Landing-page/Starbucks.webp
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Basic/Starbucks-Landing-page/Starbucks.webp:1]: Name of screenshot should be same as that of directory name. It is already mentioned in contributing guidelines.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Sneha123-zudo complete all these changes, again mentioned to me.

**Comment by Sneha123-zudo** [comment]
Sir , in readme file is it necessary to give working video ?

**Comment by Avdhesh-Varshney** [comment]
> Sir , in readme file is it necessary to give working video ?

Not necessary.

**Comment by Sneha123-zudo** [comment]
Done the changes, please check it.

File: Front-end-Projects/Basic/Starbucks-Landing-page/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+2. Clone the repository to your local machine:
+git clone https://github.com/your-username/Starbucks-Landing-page.git
+
+3. Open the project folder in your preferred code editor, now you can view website in live.
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+
+<img src="Starbucks-Landing-Page.webp">
```
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Basic/Starbucks-Landing-page/README.md:43]: image
Image is not shown. Try '/Starbucks-Landing-Page.webp' OK.
And delete Startbucks.webp file. Only 1 screenshot is sufficient.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Sneha123-zudo resolve it.

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Pull request merged**